### ISSUE TITLE
Information on Documentaion

### ISSUE THREAD

**Issue by jeamick** [fluidity/fluidity-tap-custodian-client#1]
Hello @ejwessel,

Very intestesting project !!
Does the goal is to kind of provide an interface for the custodian to integrate to this code ?

**Comment by ejwessel** [comment]
Hi @jeamick,

Thank you and great question!
The idea is to use the custodian client to pull market pricing data. This is to know what the underlying collateral is priced at, as to prevent the undercollateralization. Ideally this would be used by oracles so they know when to liquidate.

_apologies for closing and reopening. I accidentally hit the close button._

**Comment by jeamick** [comment]
Hello @ejwessel ,

Thanks for your reply.
Does basically some kinda API call to get market pricing data from the custodian interface right?

And who could be an oracle ? in this scenario is there a kind of Compensation Chamber or even an entity that can play the liquidation position role ?

**Comment by ejwessel** [comment]
@jeamick

> Does basically some kinda API call to get market pricing data from the custodian interface right?

This is correct. The API endpoints have not been published here however. In addition the code written above has only been written to one of those API's as a pilot concept.

> And who could be an oracle ? in this scenario is there a kind of Compensation Chamber or even an entity that can play the liquidation position role ?

You may want to read:
- [Our Medium article introducing the Tokenized Asset Portfolio](https://medium.com/fluidity/introducing-the-tokenized-asset-portfolio-7710e4239ab6)
- [Tokenized Asset Portfolio Whitepaper](https://tap.fluidity.io/#/)
